src/fides/api/tasks/storage.py

```diff
@@ -155,25 +166,20 @@ def upload_to_s3(  # pylint: disable=R0913
         raise ValueError(f"The parameters you provided are incorrect: {e}")
 
 
-def _handle_json_encoding(field: Any) -> Union[str, Dict[str, str]]:
```

Comment:
Moved this under `storage_util` and gave it the more descriptive name of `storage_json_encoder`.